themes/scss/map/cartodb-map-light.scss

```diff
@@ -299,8 +299,8 @@ only screen and (                min-resolution: 2dppx) {
     width:80%;
   }
   div.cartodb-zoom a {
-    background:url('../img/other@2x.png') no-repeat 0 0!Important;
-    background-size: 113px 34px!Important;
+    background:url('../img/other@2x.png') no-repeat 0 0!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>!important should not be used<br>Colon after property should be followed by one space